### ISSUE TITLE
Error running boxfuse v.1.28.0.1372 on Mac

### ISSUE THREAD

**Issue by gearoidh** [boxfuse/boxfuse-issues#177]
I get the following error when trying to run any boxfuse command:
```
$ boxfuse -u
Boxfuse client v.1.28.0.1372
Copyright 2017 Boxfuse GmbH. All rights reserved.

INTERNAL ERROR: null
```
Everything seems fine with v.1.27.1.1353 but both upgrading and fresh installs cause the INTERNAL ERROR error

**Comment by axelfontaine** [comment]
Thanks for letting us know. Could you share the output of the same command with `-X` added to also include debug output and a stacktrace?

**Comment by gearoidh** [comment]
Hi Alex,
I get the following response when I run the latest version. I've the environmental variables setup for v1.27 so am running v1.28 from the folder.

`
$ ./boxfuse -X
21:47:20.491 Boxfuse client v.1.28.0.1372
21:47:20.495 Copyright 2017 Boxfuse GmbH. All rights reserved.
21:47:20.495 
21:47:20.497 Loading configuration from /usr/local/boxfuse_latest/conf/boxfuse.conf ...
21:47:20.499 secret -> ************************************************************
21:47:20.499 user -> *******************************
21:47:20.499 Skipping non-existent config file: /Users/gearoidh/.boxfuse/boxfuse.conf
21:47:20.500 Skipping non-existent config file: /usr/local/boxfuse_latest/./boxfuse.conf
21:47:20.506 Found VBoxManage in /usr/local/bin
21:47:20.535 INTERNAL ERROR (NullPointerException): null
java.lang.NullPointerException: null
	at com.boxfuse.client.core.a.b.b.<init>(SourceFile:192)
	at com.boxfuse.client.core.Boxfuse.<init>(SourceFile:200)
	at com.boxfuse.client.commandline.Main.main(Main.java:86)

`

**Comment by axelfontaine** [comment]
Thanks! That was super helpful as we couldn't get our macOS test machines to fail. We have now fixed the issue and released 1.28.1.1375. Simply upgrade with `boxfuse -u` or if that doesn't work due to the error you reported, please redownload from either boxfuse.com or the Boxfuse Console and confirm this is now working for you.

**Comment by gearoidh** [comment]
That worked, thanks Alex
